dDatabase/CDClientDatabase/CDClientTables/CDLootTableTable.cpp

```diff
@@ -79,10 +79,10 @@ const LootTableEntries& CDLootTableTable::GetTable(uint32_t tableId) {
 
 	while (!tableData.eof()) {
 		CDLootTable entry;
-		entries[tableId].push_back(ReadRow(tableData));
+		entries.at(tableId).emplace_back(ReadRow(tableData));
```

Comment:
This is already a container with safe subscripting.  At is not necessary here and should be reverted.

Reply:
Are we intentionally keeping the possibility of it creating a new entry here?

Reply:
yes, that is the whole point of this and why i wrote it like that.